### PR TITLE
Update the databse behavior documentation

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/integrator/database.rst [hunk 1]
```diff
@@ -14,32 +15,49 @@ and one for the production, e.-g.:
...
 For the data of the ``main`` schema, we recommend to have one schema for each version of the application.
-The following example shows how an update can be performed from a version ``2017`` to a version ``2018``:
+The following example shows how an update can be performed from a version ``2019`` to a version ``2020``:
+
+========================  =============  =============
+                           Integration    Production
+                           schema name    schema name
+========================  =============  =============
+Initial state              main_2019      main_2019
+Sart a new version         main_2020      main_2019
```
Comment (jwkaltz) [inline doc/integrator/database.rst]: Start a new version

File: doc/integrator/database.rst [hunk 2]
```diff
@@ -14,32 +15,49 @@ and one for the production, e.-g.:
...
+
+Prepare the project
+~~~~~~~~~~~~~~~~~~~
 
 To be able to proceed like this, the variables ``PGSCHEMA`` and the ``DOCKER_PGSCHEMA_STATIC``
 should be managed in your makefiles:
 
-* the ``PGSCHEMA`` variable should be set in the ``Makefile``.
-* the ``DOCKER_PGSCHEMA_STATIC`` variable for the production should be set in a specific makefile for the production e.-g. ``production.mk``.
+* the ``PGSCHEMA`` variable should be set in the ``Makefile``, she will be set to ``main_2019``.
```
Comment (jwkaltz) [inline doc/integrator/database.rst]: she will be set -> in this example, it will be set

File: doc/integrator/database.rst [hunk 3]
```diff
@@ -14,32 +15,49 @@ and one for the production, e.-g.:
...
+* the ``DOCKER_PGSCHEMA_STATIC`` variable for the production should be set in a specific makefile
+  for the production e.-g. ``production.mk``, she will be set to ``integration_static`` in the
+  Mankefile, and to ``production_static`` in the ``production.mk`` file.
```
Comment (jwkaltz) [inline doc/integrator/database.rst]: Mankefile -> Makefile

File: doc/integrator/database.rst [hunk 4]
```diff
@@ -14,32 +15,49 @@ and one for the production, e.-g.:
...
+~~~~~~~~~~~~~~~~~~~
 
 To be able to proceed like this, the variables ``PGSCHEMA`` and the ``DOCKER_PGSCHEMA_STATIC``
 should be managed in your makefiles:
 
-* the ``PGSCHEMA`` variable should be set in the ``Makefile``.
-* the ``DOCKER_PGSCHEMA_STATIC`` variable for the production should be set in a specific makefile for the production e.-g. ``production.mk``.
+* the ``PGSCHEMA`` variable should be set in the ``Makefile``, she will be set to ``main_2019``.
+* the ``DOCKER_PGSCHEMA_STATIC`` variable for the production should be set in a specific makefile
+  for the production e.-g. ``production.mk``, she will be set to ``integration_static`` in the
```
Comment (jwkaltz) [inline doc/integrator/database.rst]: for the production -> for production

File: doc/integrator/database.rst [hunk 5]
```diff
@@ -14,32 +15,49 @@ and one for the production, e.-g.:
...
+Prepare the project
+~~~~~~~~~~~~~~~~~~~
 
 To be able to proceed like this, the variables ``PGSCHEMA`` and the ``DOCKER_PGSCHEMA_STATIC``
 should be managed in your makefiles:
 
-* the ``PGSCHEMA`` variable should be set in the ``Makefile``.
-* the ``DOCKER_PGSCHEMA_STATIC`` variable for the production should be set in a specific makefile for the production e.-g. ``production.mk``.
+* the ``PGSCHEMA`` variable should be set in the ``Makefile``, she will be set to ``main_2019``.
+* the ``DOCKER_PGSCHEMA_STATIC`` variable for the production should be set in a specific makefile
```
Comment (jwkaltz) [inline doc/integrator/database.rst]: for the production -> for production

File: doc/integrator/database.rst (hunk 4 above)
Comment (jwkaltz) [inline doc/integrator/database.rst]: "she will be set to" -> "it will be set for example to"

File: doc/integrator/database.rst (hunk 3 above)
Comment (jwkaltz) [inline doc/integrator/database.rst]: We now mention two times "in the production.mk" file in the same sentence, we should simplify this sentence

File: doc/integrator/database.rst [hunk 6]
```diff
@@ -14,32 +15,49 @@ and one for the production, e.-g.:
...
+  for the production e.-g. ``production.mk``, she will be set to ``integration_static`` in the
+  Mankefile, and to ``production_static`` in the ``production.mk`` file.
 * the line ``PGSCHEMA=${docker_schema}`` should be removed from your ``.env.mako`` file.
 
-Copying a schema
-----------------
-To copy a schema, `c2cgeoportal` provides a Postgres function called `clone_schema`.
+Sart a new version
```
Comment (jwkaltz) [inline doc/integrator/database.rst]: Sart -> Start

File: doc/integrator/database.rst [hunk 7]
```diff
@@ -14,32 +15,49 @@ and one for the production, e.-g.:
...
+~~~~~~~~~~~~~~~~~~
+
+When starting the changes, such as an application change and/or administration settings,
+create a new schema ``main_2020`` and use it on integration. Now, integration uses ``main_2020``,
+hile production still uses ``main_2019``.
+
+To create the new chema you should copy the old one, for that `c2cgeoportal` provides a Postgres
```
Comment (jwkaltz) [inline doc/integrator/database.rst]: To create the new chema you should copy the old one -> To create the new schema, you should copy the old one

File: doc/integrator/database.rst [hunk 8]
```diff
@@ -48,3 +66,25 @@ To use the function, connect to your database and perform the following statemen
     SELECT clone_schema(
         '<current_schema_name>',
         '<new_schema_name>', TRUE);
+
+In our case it will be:
```
Comment (jwkaltz) [inline doc/integrator/database.rst]: In our case it will be -> In our example, it will be

**Comment by sbrunner** [comment]
@jwkaltz updated :-)

**Pull request merged**